static-site/src/components/splash/index.jsx

```diff
@@ -246,13 +246,16 @@ function CardSourceIcon({ url }) {
   let maintainers, image;
 
   if (url.startsWith('/community')) {
-    maintainers = 'community members on GitHub';
+    const owner = url.split('/')[2];
```

Comment:
For narratives it's `[3]`, [e.g.](https://next.nextstrain.org/community/narratives/blab/ebola-narrative-ms/2019-09-13-sit-rep-ENGLISH)

Reply:
Thanks, fixed in 2a2e9f202be8cc9a7a4bc6f5fcd25e454f3330e4.

This made me think of something else: 32d2c7f869ed7708cfcdd5a694bfca01fe11368d

Reply:
> This made me think of something else

Nice improvement!